components/select_team/select_team.jsx

```diff
@@ -24,9 +24,11 @@ import LogoutIcon from 'components/widgets/icons/fa_logout_icon';
 
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 
+import InfiniteScroll from '../common/infinite_scroll.jsx';
+
 import SelectTeamItem from './components/select_team_item.jsx';
 
-const TEAMS_PER_PAGE = 200;
+export const TEAMS_PER_PAGE = 10;
```

Comment:
@esethna How many number of teams do you want to load here in one batch of request? 10 seems too small IMO

Reply:
Maybe let's try 30-50? Ideally we call enough that you don't see the ... and we call early enough that they are returned before you hit the bottom of the list

Reply:
On it...

Reply:
@sudheerDev made the change to 30